src/additional/accept.js

```diff
@@ -14,11 +15,23 @@ $.validator.addMethod("accept", function(value, element, param) {
 		// If we are using a wildcard, make it regex friendly
 		typeParam = typeParam.replace(/\*/g, ".*");
 
+		// Allow checking for BLANK mimetype.
+		if (typeParam.match(/^\|/) || typeParam.match(/\|\|/)) {
```

Comment:
Oooo, good catch!  I overlooked the possibility of ending the parm with a comma separator, and that should definitely have the same effect as a double-comma or a comma prefix.

Let me see if I can figure out how to revise a pull req from this web interface...